src/commands/commandConstants.ts

```diff
@@ -62,6 +62,8 @@ export namespace ClientCommandConstants {
    * Client command to execute an XML command on XML Language Server side.
    */
   export const EXECUTE_WORKSPACE_COMMAND = 'xml.workspace.executeCommand';
+
+  export const COMMAND_PALETTE_BINDING_WIZARD = 'xml.command.bind.grammar';

```

Comment:
Put this beside the `OPEN_BINDING_WIZARD` constant declaration, since they are related.